lib/minitest/reporters/junit_reporter.rb

```diff
@@ -99,16 +99,16 @@ def xml.trunc!(txt)
           txt.sub(/\n.*/m, '...')
         end
 
-        e = test.failure
+        failure = test.failure
 
         if test.skipped?
-          xml.skipped(:type => test.name)
+          xml.skipped(:type => failure.error.class.name)
```

Comment:
Could you please point in the spec where this is specified?

Reply:
The skipped element doesn't appear to be in the spec.

> name of the test looks more useful information there.

I don't see why it is useful, since it is in the test case name already.  It just seemed like it made more sense to keep it consistent with the error and failure elements.